challenges/objects-arrays.js

```diff
@@ -46,6 +70,10 @@ const graduates = [{"id":1,"first_name":"Cynde","university":"Missouri Southern
 
 Once you have the new array created, sort the universities alphabetically and log the result. */
 const universities = [];
+for (let i = 0; i < graduates.length; i++) {
+  universities.push(graduates[i].university);
+  universities.sort();
```

Comment:
you are sorting it each time with this, that's unnecessary. Just sort after the for loop